go/tao/linux_host_admin_rpc.go

```diff
@@ -47,6 +49,7 @@ func (client LinuxHostAdminClient) StartHostedProgram(path string, args ...strin
 		Args: args,
 	}
 	resp := new(LinuxHostAdminRPCResponse)
+	client.oob.ShareFDs(0, 1, 2)
```

Comment:
I know these really are constants, but for consistency with the rest of the code, let's use os.Std* instead of the bare numbers.